src/qml/DashboardView.qml

```diff
@@ -238,11 +238,226 @@ Rectangle {
         }
     }
 
+    Component {
```

Comment:
Verschieben geht zwar noch, aber nach dem Verschieben ändert sich die Größe der Kachel nicht mehr bei Events.
Das Verschieben der Event-Dashboards ist in meinen Augen auch nicht sehr intuitiv. Deswegen wäre zu überlegen, ob man das bei Events gar mehr zulässt oder man im Header noch zwei Pfeile zum Verschieben der Reihenfolge einbaut.

Reply:
Ich habe die Variante mit den Pfeilen gebaut